File: 241004/진수 to 진수/transformation-of-number-system.py
a,b = list(map(int,input().split()));
n = list(map(int,input()))
Sum = 0

for i in range(len(n)):
    Sum += a * (len(n)-1 -i) * int(n[i]);
Sum += int(n[-1])
arr = []
while True:
    if(Sum < b):
        arr.append(Sum);
        break;
    arr.append(Sum %b);
    Sum //= b;

for i in range(len(arr)-1,-1,-1):
    print(arr[i],end="");

File: 250507/진수 to 진수/transformation-of-number-system.js
const fs = require("fs");
const input = fs.readFileSync(0).toString().trim().split('\n');
const [a, b] = input[0].split(' ').map(Number);
const n = input[1].toString();

let sum = 0
const answer = []

for (let i = 0; i < n.length; i++) {
  sum = sum * 8 + Number(n[i])
}

let x = sum

while(true) {
  if (x < b) {
    answer.push(x)
    break
  }
  answer.push(x % b)
  x = Math.floor(x / b)
}

console.log(answer.reverse().join(''))